### PR TITLE
Add relativeTime support for NL

### PR DESCRIPTION
**Pull request by mahammedzkhan**
Adds relativeTime translations for NL

### CODE REVIEW

**Comment by iamkun** [comment]
@mahammedzkhan cool Thanks.

**Comment by mahammedzkhan** [comment]
Hm strange, build has failed. Can't see why though. Can you see the log? @iamkun

**Comment by iamkun** [comment]
Didn't pass the test. Try run `npm run test` on your own. I think there is a missing comma in line `ordinal`.

File: src/locale/nl.js [hunk 1]
```diff
@@ -4,7 +4,22 @@ const locale = {
   name: 'nl',
   weekdays: 'Zondag_Maandag_Dinsdag_Woensdag_Donderdag_Vrijdag_Zaterdag'.split('_'),
   months: 'Januari_Februari_Maart_April_Mei_Juni_Juli_Augustus_September_Oktober_November_December'.split('_'),
-  ordinal: n => `${n}.`
+  ordinal: n => `${n}.`,
+  relativeTime = {
```
Comment (iamkun) [inline src/locale/nl.js]: `relativeTime: { `

**Pull request merged**